### DISCUSSION TITLE
The `detail` section of the keymaps picker includes function memory addresses

### DISCUSSION THREAD

**Discussion by tristan957** [ibhagwan/fzf-lua#1538] [General]
![image](https://github.com/user-attachments/assets/e70f502c-38f6-4985-bdb3-0a7c896879fa)

Should the plugin just say `function` in this case and exclude the memory addresses.

**Comment by ibhagwan** [comment]
> Should the plugin just say function in this case and exclude the memory addresses.

I don’t think this is a big deal, perhaps even adds some value as you can see if certain mappings point to the same function, the preview btw will display the location and source of said function.

**Reply by tristan957** [reply]
Hmm, I have my doubts that users are reading memory addresses and comparing them across keymaps. Yep, the preview is nice!

Is there a way for me to override the detail formatting such that I could turn off the memory addresses for myself?
